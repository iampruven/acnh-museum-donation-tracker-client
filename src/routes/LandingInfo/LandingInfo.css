.centered{
    display: flex;
    flex-direction: column;
    text-align: center;

}

.backdrop{
    background-color: white;
    border-radius: 20px;
    border:4px gold solid;


}

@media only screen and (min-width: 900px) {
    .backdrop{
        background-color: white;
        border-radius: 20px;
        border:4px gold solid;

    
    }
  }