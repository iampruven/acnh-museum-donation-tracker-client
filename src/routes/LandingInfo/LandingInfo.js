import React from "react";
import './LandingInfo.css'
export default class LandingInfo extends React.Component {
  render() {
    return (
      <section className="centered backdrop">
        <div>
          <h2>
            Welcome to Animal Crossing New Horizons: Museum Donation Tracker
          </h2>
        </div>
        <div className="para">
          <p>Please register or login to view contents.</p>
          <p>
            Registered users may keep track of all donated items to their museum
            by adding Fish, Fossils, Bugs and view them per category.
          </p>
          <p>Demo user:</p>
          <p>username: demouser</p>
          <p>password: 123456!Du</p>
        </div>
      </section>
    );
  }
}
