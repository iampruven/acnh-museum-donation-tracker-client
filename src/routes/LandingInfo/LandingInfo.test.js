import React from 'react';
import ReactDOM from 'react-dom';
import LandingInfo from './LandingInfo';
import renderer from 'react-test-renderer';

//smoke test
describe('renders without crashing',()=>{
    it('it renders without crashing',()=>{
        const div = document.createElement('div');

        ReactDOM.render(<LandingInfo />,div);
        ReactDOM.unmountComponentAtNode(div);
    })


//snapshot test
    it('renders the UI as expected', ()=>{
        const tree = renderer
            .create(<LandingInfo />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    })



})