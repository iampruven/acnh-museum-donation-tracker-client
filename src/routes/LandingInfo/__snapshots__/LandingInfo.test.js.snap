// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing renders the UI as expected 1`] = `
<section
  className="centered backdrop"
>
  <div>
    <h2>
      Welcome to Animal Crossing New Horizons: Museum Donation Tracker
    </h2>
  </div>
  <div
    className="para"
  >
    <p>
      Please register or login to view contents.
    </p>
    <p>
      Registered users may keep track of all donated items to their museum by adding Fish, Fossils, Bugs and view them per category.
    </p>
    <p>
      Demo user:
    </p>
    <p>
      username: demouser
    </p>
    <p>
      password: 123456!Du
    </p>
  </div>
</section>
`;
